### PR TITLE
Add information on GCF wrappers

### PR DESCRIPTION
**Pull request by tmorrisonsfx**
Adding information on wrappers for Node.js, Go, and Python

Jira issue: https://signalfuse.atlassian.net/browse/DOCS-1238
Docs URL: https://docs.signalfx.com/en/latest/integrations/integrations-reference/integrations.google.cloud.functions.html

### CODE REVIEW

**Comment by tmorrisonsfx** [comment]
@mstumpfx Hi Mark, can you review this PR? I am specifically looking for the following feedback:

1. This PR is based on the AWS Lambda wrappers content. I know some of the information will differ for GCF, so please review the  SignalFx Wrapper vs CloudWatch monitoring section.
2. What are the metrics for the SignalFx Wrapper for GCF? 
3. Is there a unique ID for GCF? Is it `gcf_project_id`?

File: google-cloud-functions/README.md [hunk 1]
```diff
@@ -31,6 +39,42 @@ To access this integration, [connect to Google Cloud Platform](https://docs.sign
 
 None
 
+### Wrappers
+
+#### SignalFx Wrapper vs CloudWatch monitoring
```
Comment (mstumpfx) [inline google-cloud-functions/README.md:44]: Google's cloud monitoring service is called Google Cloud Monitoring (rebranded from Google StackDriver, but I don't think we have changed the branding in our own documentation yet).
CloudWatch is specific to Amazon

File: google-cloud-functions/README.md [hunk 2]
```diff
@@ -31,6 +39,42 @@ To access this integration, [connect to Google Cloud Platform](https://docs.sign
...
+
+##### Step 1: Review and Select a Deployment Method
+
+While you can use both the SignalFx Wrapper and CloudWatch methods to monitor functions at the same time, you also have the option to use only one method.
```
Comment (mstumpfx) [inline google-cloud-functions/README.md:48]: change `CloudWatch`

File: google-cloud-functions/README.md [hunk 3]
```diff
@@ -31,6 +39,42 @@ To access this integration, [connect to Google Cloud Platform](https://docs.sign
...
+
+Before you select a single method (or both), consider the following statements:
+
+- Each method sends different metrics and uses different properties to uniquely identify Cloud Functions.
+
+- CloudWatch data is reported with a delay; the delay can range from 1 to 10 minutes. The SignalFx Wrapper sends data immediately when it becomes available.
```
Comment (mstumpfx) [inline google-cloud-functions/README.md:54]: change `CloudWatch`

File: google-cloud-functions/README.md [hunk 4]
```diff
@@ -31,6 +39,42 @@ To access this integration, [connect to Google Cloud Platform](https://docs.sign
...
+
+- CloudWatch data is sent automatically for all Cloud Functions, whereas the SignalFx Wrapper needs to be added to each function.
```
Comment (mstumpfx) [inline google-cloud-functions/README.md:56]: change `CloudWatch`

File: google-cloud-functions/README.md [hunk 5]
```diff
@@ -31,6 +39,42 @@ To access this integration, [connect to Google Cloud Platform](https://docs.sign
...
+
+Review the installation methods for each Cloud Function wrapper:
+
+- <a href="https://github.com/signalfx/serverless-go/blob/master/gcfwrapper/README.md" target="_blank">Go</a>
+- <a href="https://github.com/signalfx/serverless-nodejs/blob/master/gcf-wrapper/README.md" target="_blank">Node.js</a>
+- <a href="https://github.com/signalfx/serverless-python/blob/master/signalfx_gcf/README.rst" target="_blank">Python</a>
+
+*Option 2: Use CloudWatch*
+
+By default, SignalFx imports all CloudWatch metrics that are available in your account. To retrieve metrics for a subset of available services or regions, you need to modify the connection on the Integrations page in the SignalFx UI.
```
Comment (mstumpfx) [inline google-cloud-functions/README.md:74]: change `CloudWatch`

File: google-cloud-functions/README.md [hunk 6]
```diff
@@ -31,6 +39,42 @@ To access this integration, [connect to Google Cloud Platform](https://docs.sign
...
+
+For more information, see <a href="https://github.com/signalfx/integrations/tree/master/aws">connect to CloudWatch</a>.
```
Comment (mstumpfx) [inline google-cloud-functions/README.md:76]: change link to google stackdriver integration and change `CloudWatch`

**Comment by mstumpfx** [comment]
> @mstumpfx Hi Mark, can you review this PR? I am specifically looking for the following feedback:
> 
> 1. This PR is based on the AWS Lambda wrappers content. I know some of the information will differ for GCF, so please review the  SignalFx Wrapper vs CloudWatch monitoring section.
I left some comments in the PR.

> 2. What are the metrics for the SignalFx Wrapper for GCF?
`function.invocations`, `function.errors`, `function.duration`, `function.cold_starts`

> 3. Is there a unique ID for GCF? Is it `gcf_project_id`?
The wrapper does not emit a single unique ID for a function, however the subset of dimensions is probably unique. Those include the project id, function name, function version, region, and wrapper version.

**Comment by tmorrisonsfx** [comment]
Closing this PR due to issues with rebasing and deleting commits. Work is continued in https://github.com/signalfx/integrations/pull/869.

**Pull request closed without merging**